src/services/Store/store/account.slice.ts

```diff
@@ -96,6 +97,29 @@ export const getAccounts = createSelector([getAppState], (s) => {
     }))
   }));
 });
+
+export const selectCurrentAccounts = createSelector(
+  [getAccounts, getFavorites],
+  (accounts, favorites) => {
+    return accounts
+      .filter((a) => a && 'uuid' in a)
+      .filter(({ uuid }) => favorites.indexOf(uuid) >= 0);
+  }
+);
+
+export const selectAccountTxs = createSelector([getAccounts], (accounts) => {
```

Comment:
Is there a reason behind the change of naming convention? Other selectors are named as 'getters'

Reply:
Imitating the redux examples.
It creates a clear separation between the selector and an api `get` method.